docs/src/pages/components/data-grid/components/CustomColumnMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import { makeStyles } from '@material-ui/core/styles';
+import * as React from 'react';
```

Comment:
Convention in the demos

```suggestion
import * as React from 'react';
import { makeStyles } from '@material-ui/core/styles';
```

x the others